conn_go18.go

```diff
@@ -76,6 +76,19 @@ func (cn *conn) BeginTx(ctx context.Context, opts driver.TxOptions) (driver.Tx,
 	return tx, nil
 }
 
+func (cn *conn) Ping(ctx context.Context) error {
```

Comment:
The godoc for the Ping function (https://golang.org/pkg/database/sql/driver/#Pinger) says it should return `driver.ErrBadConn`. I would like that error to be explicitly returned instead of returning whatever it is simpleQuery and rows.Close return (which I think is indeed driver.ErrBadConn from a quick look). We should hard code it here because that's what the API requires).

Reply:
Will update this tonight. Thanks.

Reply:
This should be solved with latest commit.